roles/nginx/tasks/ssl.yml

```diff
@@ -28,10 +25,21 @@
     state: stopped
   when: not fullchain.stat.exists
 
-- name: generate ssl certificates for {{ domain }}
-  raw: certbot certonly --standalone --email {{ email }}
-       --agree-tos -n -d {{ domain }},www.{{ domain }}
-  when: not fullchain.stat.exists
+
+- name: "generate staging ssl certificates for {{ domain }}"
+  command: "certbot certonly --standalone --email {{ email }}
+       --agree-tos -n -d {{ domain }},www.{{ domain }} --staging"
+  when:
+    - not fullchain.stat.exists
+    - ssl == 'staging'
```

Comment:
can we do jinja templating in the tasks definition ? I thought it was only for templates.
I'll test that